--- a/beam-postgres.cabal	2020-12-08 10:43:19.785670819 +0200
+++ b/beam-postgres.cabal	2020-12-08 10:43:11.686659624 +0200
@@ -50,7 +50,7 @@
                       monad-control        >=1.0  && <1.1,
                       mtl                  >=2.1  && <2.3,
                       conduit              >=1.2  && <1.4,
-                      aeson                >=0.11 && <1.5,
+                      aeson                >=0.11 && <1.6,
                       uuid-types           >=1.0  && <1.1,
                       case-insensitive     >=1.2  && <1.3,
                       scientific           >=0.3  && <0.4,
